corehq/apps/reports/tasks.py

```diff
@@ -204,24 +211,72 @@ def apps_update_calculated_properties():
 
 
 @task(bind=True, default_retry_delay=15 * 60, max_retries=10, acks_late=True)
-def send_email_report(self, recipient_list, body, subject, report_class, report_state):
+def send_email_report(self, recipient_emails, domain, report_slug, report_type, request_data, once, cleaned_data):
```

Comment:
oof - this is a lot more involved than I was expecting - sorry for sending you down this path.

Reply:
yes It involved a little more. But its cool, I got to learn more bits of code. 
#CheersToThat 👍